src/checkBoxEvent.js

```diff
@@ -0,0 +1,13 @@
+export default (t) => {
```

Comment:
The user is unable to see items that he/she can check to mark them complete.  Please add some items in the items array and make sure they are shown on the page. Moreover, the user is able to check the off from the list.
Your page:
![list](https://user-images.githubusercontent.com/35504149/144260317-5ba2dead-e921-4d20-81eb-1de5bd3c6b66.png)

Reply:
- [ ] Good Job

![done](https://user-images.githubusercontent.com/55635977/144267139-eebcb852-01d4-4bf1-972d-5bef14a06a35.png)